packages/graphback-datasync/src/DataSyncPlugin.ts

```diff
@@ -64,6 +64,20 @@ export class DataSyncPlugin extends GraphbackPlugin {
                     lastSync: GraphQLNonNull(GraphQLString),
                     filter: filterTypeName
                 })
+
+                // Add updatedAt arg to update and delete input types
+                // for conflict resolution
+                const {fieldNames} = metadataMap;
+                [GraphbackOperationType.DELETE, GraphbackOperationType.UPDATE].map((operation: GraphbackOperationType) => {
```

Comment:
we do not need an array here since this call https://github.com/aerogear/graphback/pull/1467/files#diff-37ea145aded7cc3fa4e8f167fac3af2bR72 will return the same inputType because of https://github.com/aerogear/graphback/blob/eec1d8bfc6395311c8c61959ade0c9dddf86cb34/packages/graphback-core/src/crud/mappingHelpers.ts#L62-L64

Reply:
Good point, thanks!